lib/bookingsync-engine.rb

```diff
@@ -1 +1,29 @@
-require 'bookingsync'
+require "bookingsync"
+
+module BookingSyncEngine
+  cattr_accessor :support_multi_applications
+  self.support_multi_applications = false
+
+  cattr_accessor :single_app_model
+  self.single_app_model = -> { ::Account }
+
+  cattr_accessor :multi_app_model
+  self.multi_app_model = -> { ::Account }
+
+  def self.setup
+    yield self
+  end
+
+  def self.support_multi_applications?
```

Comment:
why do we need it?

Reply:
Hmm, ok, it's used in engine config. Why driving it via ENV var though and not a simple attribute? ENV var suggests that we might want to have it enabled e.g. on staging but disabled on production. Would that be the case here?

Reply:
Indeed. The ENV var doesn't seem to have some added value.